tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -55,41 +55,48 @@ def _deterministic_libtool_flags(ctx):
         return ["-D"]
     return []
 
+def _target_os_version(ctx):
+    platform_type = ctx.fragments.apple.single_arch_platform.platform_type
+    xcode_config = ctx.attr._xcode_config[apple_common.XcodeVersionConfig]
+    return xcode_config.minimum_os_for_platform_type(platform_type)
+
 def _impl(ctx):
+    target_os_version = _target_os_version(ctx)
```

Comment:
I guess can you split out the original change you had here around `s/arch/target` and the potentially replacing version_min? the latter seems harder

Reply:
OK I'll keep this change to just replacing arch to target.

Reply:
https://github.com/bazelbuild/bazel/pull/14898